### ISSUE TITLE
Storage Tablet Dimension UI

### ISSUE THREAD

**Issue by Master0r0** [McJtyMods/RFTools#1848]
Issue description: 
Server Issue
Attempting to use a Storage Tablet with a Storage Scanner module in other dimensions results in the UI not reading it's settings and not storing them when changed.
In the nether accessing my tablet results in the wide storage list toggle resetting to default.
In the betweenlands the same thing happens additionally the power bar is at 0
Additionally, while the tablet is open, the server log is spammed with 
[12:22:47] [Server thread/INFO] [mcjty.lib.varia.Logging]: createStartScanPacket: TileEntity is not a CommandHandler!
But no other related output is logged

Steps to reproduce: 
Set Settings of tablet such as shrinking the storage list.
Travel to another dimension

Versions:
* Minecraft: 1.12.2
* Forge: 14.23.4.2760
* McJtyLib: 1.12-3.0.5
* RFTools: 1.12-7.56

Possibly incompatible mods and versions: 
FoamFix: 0.10.1-1.12.2

**Comment by McJty** [comment]
Is the storage scanner chunkloaded?

**Comment by Master0r0** [comment]
Yes it is, loaded with both FTB Utilities and Extra Utilities 2's chunk loading solutions

**Comment by josephcsible** [comment]
Can you reproduce this with no mods installed except for RFTools, McJtyLib, and a chunk loader (preferably a mod that is just a chunk loader, like ChickenChunks)?

**Comment by Master0r0** [comment]
It does still seem to occur with only McJtyLib, RFTools and Wierding Gadget as a chunkloader.
Here is my latest.log just incase I'm missing something: https://pastebin.com/7EDPy1F3

**Comment by Master0r0** [comment]
So the issue occurs even in the same dimension, I was experiencing it when I was only 8 chunks away, well within visual range of the system, but going a chunk closer seemed to make it work fine again
